### PR TITLE
Basic cleanup

### PR DESCRIPTION
**Pull request by brandonsturgeon**

### CODE REVIEW

File: lua/cfc_gmod_scripts/init.lua [hunk 1]
```diff
@@ -7,13 +7,14 @@ local function log( name )
...
+    if prefix == "sv_" then
+        if CLIENT then return end
         return include( path )
     end
 
-    if prefix == "cl_" and CLIENT then
-        AddCSLuaFile( path )
-        return include( path )
+    if prefix == "cl_" then
+        if SERVER then return AddCSLuaFile( path ) end
```
Comment (wrefgtzweve) [inline lua/cfc_gmod_scripts/init.lua:16]: image

File: lua/cfc_gmod_scripts/init.lua [hunk 2]
```diff
@@ -25,10 +26,12 @@ local _, dirs = file.Find( topLevel .. "/*", "LUA" )
...
 for _, dir in ipairs( dirs ) do
     log( dir )
+
     local files = file.Find( topLevel .. "/" .. dir .. "/*", "LUA" )
     for _, fil in ipairs( files ) do
         local prefix = string.sub( fil, 1, 3 )
         local path = dir .. "/" .. fil
-        realmInclude( prefix, path )
+
+        ProtectedCall( function() realmInclude( prefix, path ) end )
```
Comment (wrefgtzweve) [inline lua/cfc_gmod_scripts/init.lua:35]: not needed, include doesn't halt when it errors

**Review by wrefgtzweve — COMMENTED** [review]
Some small things i saw, should be good otherwise

File: lua/cfc_gmod_scripts/init.lua (hunk 1 above)
Comment (brandonsturgeon) [reply lua/cfc_gmod_scripts/init.lua:16]: Sure but I think that's a trick useful to `include` a file that has `AddCSLuaFile()` at the top.

Anyway, primarily, this code change makes sure the server isn't also trying to include client files

**Pull request closed without merging**